examples/vault/package.json

```diff
@@ -2,13 +2,10 @@
   "name": "vault_examples",
   "version": "2.0.0",
   "description": "Pangea Vault examples",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "private": true,
   "author": "Pangea",
   "license": "MIT",
   "dependencies": {
-    "pangea-node-sdk": "3.5.0"
+    "pangea-node-sdk": "file:../../packages/pangea-node-sdk"
```

Comment:
How do you want to handle it? Next version should be 3.6.0

Reply:
This approach will be better once coupled with CI for the examples, as it'll catch incoming changes that break examples.